test/users/bachman/Chap2D/compare_fortran.chpl

```diff
@@ -0,0 +1,46 @@
+use IO;
```

Comment:
Are the contents of this file actually used anywhere in these tests?

Reply:
No I don't think so.  I use that module for sanity checking.